test/unit/volume_manager_btrfs_test.py

```diff
@@ -6,7 +6,10 @@
 from textwrap import dedent
 import datetime
 
-from .test_helper import *
+from .test_helper import (
+    raises, patch_open
+)
```

Comment:
Parenthesis would be correct if you would:
```python
from .test_helper import (
    raises,
    patch_open
)
```
Sure you can do that too, but [don't](https://medium.com/@webseanhickey/the-evolution-of-a-software-engineer-db854689243#.qxgpwbfak).